lib/modules/stats/country_statistics_api.rb

```diff
@@ -88,14 +97,18 @@ def get_global_stats(endpoint=nil)
 
     # This is only used for global stats
     def format_data(data, endpoint)
-      json = { title: ENDPOINTS[endpoint.to_sym][:name], charts: [] }
+      json = { title: ATTRIBUTES[endpoint.to_sym][:name], charts: [] }
       chart_json = Aichi11Target::DEFAULT_CHART_JSON.dup
-      field = ENDPOINTS[endpoint.to_sym][:field]
-
-      _sum = data.inject(0) do |sum, x|
-        sum + (x[field] ? x[field] : 0)
+      attribute = ATTRIBUTES[endpoint.to_sym][:attribute]
+
+      attr_area_sum = total_area_sum = 0
+      data.map do |x|
+        attr_area = (x[attribute] ? x[attribute] : 0)
+        total_area = x[COUNTRY_AREA_ATTRIBUTE] || 0
+        attr_area_sum += (attr_area > 0 && total_area > 0) ? (attr_area * total_area / 100) : 0
```

Comment:
could either `attr_area` or `total_area` be negative?? If this is the case the check is correct, but if they can only have `0` or positive values I think you are safe even without this ternary operator